source/index.html

```diff
@@ -1,12 +1,137 @@
+        </p>
+        <ul class="features__list">
+          <li class="features__item">
+            <div class="features__card features__card--main">
+              <h3 class="features__title">Настоящий городок</h3>
+              <p class="features__number">-№1-</p>
+              <p class="features__description">Седона не атракцион для туристов, там течет своя жизнь</p>
+            </div>
+            <ul class="advantages-list">
+              <li class="advantages-list__item advantages-lis__item--living">
+                <h3 class="advantages-list__title ">Жилье</h3>
+                <p class="advantages-list__description">Рекомендуем пожить в настоящем мотеле, все как в кино!</p>
+              </li>
+              <li class="advantages-list__item advantages-list__item--food">
+                <h3 class="advantages-list__title">Еда</h3>
+                <p class="advantages-list__description">Всегда заказывайте фирменный бургер, Вы не разочаруетесь!</p>
+              </li>
+              <li class="advantages-list__item advantages-list__item--souvenirs">
+                <h3 class="advantages-list__title">Сувениры</h3>
+                <p class="advantages-list__description">Не только китайского, но и местного производства!</p>
+              </li>
+            </ul>
+          </li>
+          <li class="features__card features__card--main">
+            <h3 class="features__title">Там есть мост дьявола</h3>
+            <p class="features__number">-№2-</p>
+            <p class="features__description">Да, по нему можно пройти! Если Вы осмелитесь, конечно</p>
+          </li>
+          <li class="features__card">
+            <h3 class="features__title">Небольшая площадь</h3>
+            <p class="features__number">-№3-</p>
+            <p class="features__description">Все интересные места находятся очень близко</p>
+          </li>
+          <li class="features__card">
+            <h3 class="features__title">Красивая дорога</h3>
+            <p class="features__number">-№4-</p>
+            <p class="features__description">Ехать в Седону из Лас-Вегаса совсем не скучно!</p>
+          </li>
+          <li class="features__card">
+            <h3 class="features__title">Мало туристов</h3>
+            <p class="features__number">-№5-</p>
+            <p class="features__description">Большинство едет в Гранд Каньон и толпится там</p>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="hostel-searching">
+      <h2 class="visually-hidden">
+        Поиск жилья
+      </h2>
+      <div class="hostel-searching__wrapper">
+        <h3 class="hostel-searching__title">Заинтересовались?</h3>
+        <p class="hostel-searching__description">
+          Укажите предполагаемые даты поездки, и мы покажем Вам лучшие предложения гостиниц в Седоне</p>
+        <a class="hostel-searching__button" href="" target="blank" type="button">Найти гостиницу</a>
```

Comment:
`target="blank"` обычно используется, когда ссылка ведет на сторонний ресурс, здесь это не тот случай.
```suggestion
        <a class="hostel-searching__button" href="">Найти гостиницу</a>
```